MetaMask/metamask-extension
# [FLASK] Fix overflow issues with text coming from snap UI

## Explanation
Fixes a few overflow issues with text coming from snap UI, by wrapping instead of allowing overflow.

Fixes #18168 

## Screenshots/Screencaps



### Before
<img width="361" alt="Screenshot 2023-03-15 at 14 55 35" src="https://user-images.githubusercontent.com/1561200/225330811-f70552cb-f097-4977-ba8d-bb13911302ec.png">



### After
<img width="363" alt="Screenshot 2023-03-15 at 14 40 53" src="https://user-images.githubusercontent.com/1561200/225334205-1138f951-4ead-4848-a8a6-0a8d9eeb33fe.png">



## Manual Testing Steps
1. Create a snap using `text()` from `@metamask/snaps-ui`
2. Have the string inside `text()` be one very long word
3. View your snap UI and check that it doesn't overflow
